tools/ffi_coverage/src/allowlist.rs

```diff
@@ -264,9 +264,11 @@ lazy_static::lazy_static! {
         "icu::normalizer::ComposingNormalizer::normalize_utf16",
         "icu::normalizer::ComposingNormalizer::normalize_utf16_to",
         "icu::normalizer::ComposingNormalizer::is_normalized_utf16",
+        "icu::normalizer::ComposingNormalizer::is_normalized_utf16_up_to",
```

Comment:
please expose these, using `&DiplomatStr16`

Reply:
In the latest changeset

Reply:
Thanks, don't forget to run `cargo make diplomat-gen`